o!AOT: Round of 16 (#2418)
ppy/osu-wiki: wiki/Tournaments/o!AOT/1/en.md

---
tags:

> - osu! Asia Oceania Tournament

Reviewer: ```suggestion
```
words in titles don't need to be tagged

- o!AOT
---
# osu! Asia Oceania Tournament

![o!AOT Logo](img/logo.png)

The **osu! Asia Oceania Tournament** (***o!AOT***) is an Asian and Oceanian double-elimination team-based osu! tournament hosted by ![][flag_ID] [Fuma](https://osu.ppy.sh/users/1501956) and ![][flag_ID] [nya10](https://osu.ppy.sh/users/1372608). It is the first installment of the osu! Asia Oceania Tournament.

## Tournament Schedule

| Event | Timestamp |
| --: | :-- |
| Registration Phase | 2019-06-10/2019-07-07 |
| Live Drawings | 2019-07-20 (19:00 UTC+7) |
| Qualifiers Stage | 2019-07-26/2019-07-28 |
| Group Stage | 2019-08-02/2019-08-11 |
| Round of 16 | 2019-08-16/2019-08-18 |
| Quarterfinals | 2019-08-23/2019-08-25 |
| Semifinals | 2019-08-30/2019-09-01 |
| Finals | 2019-09-06/2019-09-08 |
| Grand Finals | 2019-09-13/2019-09-15 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Unique profile badge, 2 months of osu!supporter tag

Reviewer: missing `|` at the end

| D | **Bin Boys** | **5** | 3 | Nov 14th | [#1](https://osu.ppy.sh/community/matches/53995030) |
| H | **進撃のバブルティー** | **5** | 0 | bus girls | [#1](https://osu.ppy.sh/community/matches/53995469) |
| G | **과로사** | **5** | 1 | Hot Loli(s) | [#1](https://osu.ppy.sh/community/matches/53996389) |
| G | **okguysweneedaname** | **5** | 2 | 艾莉皮肤店 | [#1](https://osu.ppy.sh/community/matches/53997555) |
| A | 黃建智 | 4 | **5** | **reyuza ganteng** | [#1](https://osu.ppy.sh/community/matches/53999327) |

---

## Ruleset

### Qualifier Information

1. Scoring will be done by using the following [formula](http://bit.ly/2HNYw7M).
2. Every map in the qualifier mappool will have to be played once.
3. Notify `@host` to reschedule your lobby in `#reschedule` that can be found in o!AOT's Discord server using the given format.
4. Team placement in the qualifiers will determine seedings in group stage as follows:
   - Seed 1 - 8 will be identified as a **top seed team.**
   - Seed 9 - 16 will be identified as a **high seed team.**
   - Seed 17 - 24 will be identified as a **mid seed team.**
   - Seed 24 - 32 will be identified as a **low seed team.**

> 5. Each group in group stage is consist of: 1 top seed, 1 high seed, 1 mid seed, and 1 low seed team.

Reviewer 2: ```suggestion
5. Each group in group stage consists of: 1 top seed, 1 high seed, 1 mid seed, and 1 low seed team.
```

6. However, for top seed team, their group will be placed as follows:
   - Seed 1 will be placed in Group A
   - Seed 2 will be placed in Group H
   - Seed 3/4 will be placed in Group D/E
   - Seed 5/6/7/8 will be placed in Group B/C/F/G
7. A team will have **2 minutes** to switch their roster(s) between each map.
   - After the given time expires, the referee will start the map.

> 8. If a team get a technical difficulty in the middle of a qualifier map, the following team gets a chance to replay that specific map. (only once per team in the whole qualifier playthrough)

Reviewer 2: ```suggestion
8. If a team gets a technical difficulty in the middle of a qualifier map, the following team gets a chance to replay that specific map. (only once per team in the whole qualifier playthrough)
```

### Match Procedures

1. Teams will use `!roll` tournament to determine the order of picking and banning beatmaps.

> 2. The team that scores higher will pick the first beatmap to play and second to bann. (start picking after the second bann is mentioned)

Reviewer 2: ```suggestion
2. The team that scores higher will pick the first beatmap to play and second to ban. (start picking after the second ban is mentioned)
```

3. The team that scores lower will start banning the map first and picking map to play second.
4. For Grand Finals, there will be no bracket reset. Instead, the team in winner bracket will be given a choice to win/lose the `!roll`, one point advantage and a free ban. Their pick/ban order can follow the #2 and #3 point of this section.
5. A team will be given **2 minutes** to pick/ban a map
6. A team will be given **2 minutes** to change their roster(s).
7. If the team does not have their pick/ban within the given time, the pick/ban will be randomized by `!roll x` (where x is the amount of maps left, tiebreaker map is excluded).
8. Disconnection rules (timer counts when the map begins):
   - First 30 seconds: Rematch the map.
   - After 30 seconds: No rematch.

>    - If any disconnection happens, the team who has disonnected member should not wait for their member. However in the case there are only 2 members in the said team, both teams should wait until 5 minutes before it is declared a loss to the disconnecting team.

Reviewer 2: ```suggestion
   - If any disconnection happens, the team who has disconnected member should not wait for their member. However in the case there are only 2 members in the said team, both teams should wait until 5 minutes before it is declared a loss to the disconnecting team.
```
